src/util/translation.h

```diff
@@ -16,13 +16,26 @@
 struct bilingual_str {
     std::string original;
     std::string translated;
+
+    bilingual_str& operator+=(const bilingual_str& rhs)
+    {
+        original += rhs.original;
+        translated += rhs.translated;
+        return *this;
+    }
+
+    bilingual_str& operator+=(const std::string& chars)
```

Comment:
I think it would be safer to not allow this and force the caller to go through `Untranslated`

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/18922#issuecomment-626197278).